cv: Extract more metadata from SVG (#140)
echemdb/svgdigitizer: doc/news/cvmetadata2.rst

**Added:**

> * Added properties to extract `tags` describing the data, `figure_label` and `curve_label` from the SVG, which are included in the property `metadata`.

Reviewer: ```suggestion
* Added property `CV.tags`, `CV.figure_label`, and `CV.curve_label` which are extracted from `<text>` elements in the SVG. The extracted values are stored in `CV.metadata`.
```

I didn't understand what you did until I read the code. Maybe it's easier to understand now?

Author: Looks good to me.

**Changed:**

> * Changed entry `axes` in `figure description` and `data description` to `field` according to `Frictionless Table Schema <https://specs.frictionlessdata.io/table-schema>`_.

Reviewer: ```suggestion
* Changed entry `axes` in `figure description` and `data description` to `fields` according to `Frictionless Table Schema <https://specs.frictionlessdata.io/table-schema>`_.
```

> * Changed entry `axes` in `figure description` and `data description` to `field` according to `Frictionless Table Schema <https://specs.frictionlessdata.io/table-schema>`_ and update output json in CLI option `cv` according to this scheme.

Reviewer: ```suggestion
* Changed entry `axes` in `figure description` and `data description` to `field` according to `Frictionless Table Schema <https://specs.frictionlessdata.io/table-schema>`_.
```
